python/tvm/tir/schedule/schedule.py

```diff
@@ -71,6 +73,13 @@ def __init__(self) -> None:
 }
 
 
+class BufferType(enum.IntEnum):
+    """Type of buffer in access regions of a block"""
+
+    READ = 0
+    WRITE = 1
+
```

Comment:
@vinx13 Sorry I'm late and missed the code review. On this particular change, I'm 100% in favor of having enum type on C++ side; However, it could be more to "just use string" on the python side, i.e. use "read" and "write" to indicate BufferType. If you agree with my opinion, would you mind sending a quick patch? Thanks a lot!

Reply:
This sounds good to me as well, and would give the same readability benefits at the caller side.  I kind of like the `enum` usage in python as well, since it lets the reader know what options are possible, but that's the less important benefit.

At some point, it would be nice to have a macro to define an enum in C++, along with its value/name mapping and an FFI interface, so that there would be a clear way to handle these.